### PR TITLE
fix: ack in batches

### PR DESCRIPTION
**Pull request by chasers**

### CODE REVIEW

File: lib/logflare/source/bigquery/buffer_producer.ex [hunk 1]
```diff
@@ -33,13 +33,8 @@ defmodule Logflare.Source.BigQuery.BufferProducer do
...
   def ack(source_id, successful, unsuccessful) when is_atom(source_id) do
-    Enum.each(successful, fn %{data: %LE{}} = message ->
-      BufferCounter.ack(source_id, message.data.id)
-    end)
-
-    Enum.each(unsuccessful, fn %{data: %LE{}} = message ->
-      BufferCounter.ack(source_id, message.data.id)
-    end)
+    BufferCounter.ack(source_id, successful)
+    BufferCounter.ack(source_id, unsuccessful)
```
Comment (Ziinc) [inline lib/logflare/source/bigquery/buffer_producer.ex:37]: Doing `BufferCounter.ack(source_id, successful ++ unsuccessful)` would work too. or `BufferCounter.ack(source_id, {successful, unsuccessful})` if splitting the counts has value in future

**Review by Ziinc — APPROVED** [review]

**Comment by Ziinc** [comment]
Screenshot 2023-07-20 at 3 11 53 AM

**Comment by chasers** [comment]
Huh, I'm using it in a function head match.

File: lib/logflare/source/bigquery/buffer_producer.ex (hunk 1 above)
Comment (chasers) [reply lib/logflare/source/bigquery/buffer_producer.ex]: Good call, changed.

**Pull request merged**